exo-run/src/service-restarter.ls

```diff
@@ -30,19 +31,21 @@ class ServiceRestarter extends EventEmitter
 
   _restart: ->
     @watcher.close!
-    cwd = @docker-config-location
-    DockerCompose.kill-container {service-name: @role, cwd, @write}, (exit-code) ~>
-      | exit-code => return @emit 'error', "Docker failed to kill container #{@role}"
-      @write "Docker container stopped"
-
-      DockerCompose.create-new-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-        | exit-code => return @emit 'error', "Docker image failed to rebuild #{@role}"
-        @write "Docker image rebuilt"
-
-        DockerCompose.start-container {service-name: @role, cwd, @env, @write}, (exit-code) ~>
-          | exit-code => return @emit 'error', "Docker container failed to restart #{@role}"
-          @watch!
-          @logger.log {role: 'exo-run', text: "'#{@role}' restarted successfully"}
+    set-timeout ( ~>
```

Comment:
I also think the delay time can drop to 100 milliseconds. We are mainly after saving multiple files at the same time.